content/pages/2_functionalities.md

```diff
@@ -126,8 +126,17 @@ Read the documentation:
 * MapServer
 * QGIS Server
 * ArcGIS for Server
+* Fulltext search service
```

Comment:
I won't put this here, it was more to mention which apps are supported as map server.

Reply:
```suggestion
```